app/components/modal-catalog-launch/template.hbs

```diff
@@ -77,7 +77,20 @@
                   {{#if (eq question.type "multiline")}}
                     {{textarea value=question.answer rows="5" class="form-control"}}
                   {{else}}
-                    Unknown question type: {{question.type}}
+                    {{#if (eq question.type 'service')}}
+                      {{#if (not loadingServices)}}
```

Comment:
@vincent99  I need to delay the loading of the select template until we've gotten/parsed all services so the default is selected if there is one. I tried a couple different things (show loading spinner around everything, loading only on questions) but they all seemed a little janky cause the whole modal moves around. With this only the select will pop in after and its a little less janky but still not ideal.